main.ml

```diff
@@ -1087,7 +1212,24 @@ let description = String.concat ~sep:" " [
   "the scripts inside.";
 ]
 
+type backend_error = string * (string list) [@@deriving sexp]
+
+(** registers pretty printers for `Backend_error*` exceptions.
+ *  Otherwise we only log Backend_error_with_backtrace(_), with the pretty
+ *  printer we can log the actual arguments of the exception.
+ * *)
+let register_exn_pretty_printers () =
```

Comment:
Is this `Sexplib` from `xen-api-libs-transitional`? If that is the case, can we use some other upstream `sexp` parser instead? If that is not the case, please ignore my comment

Reply:
It should be sexplib from Janestreet/Core.